src/js/components/FragmentList/fragmentList.jsx

```diff
@@ -16,43 +16,9 @@
 
 import React, { useState } from 'react';
 import { useSelector } from 'react-redux';
-import styled from 'styled-components';
+import Styled from './fragmentList.style';
 import Fragment from './fragment';
```

Comment:
I would consider using different name as this clashes with internal React.Fragment